worlds/peaks_of_yore/__init__.py

```diff
@@ -63,6 +63,12 @@ def generate_early(self) -> None:
         if self.options.start_with_barometer:
             self.multiworld.push_precollected(self.create_item("Barometer"))
 
+        if self.options.start_with_chalk:
+            self.multiworld.push_precollected(self.create_item("Chalkbag"))
+
+        if self.options.start_with_coffee:
+            self.multiworld.push_precollected(self.create_item("Coffee Unlock"))
```

Comment:
there's a helper function on the World
```suggestion
        if self.options.start_with_chalk:
            self.push_precollected(self.create_item("Chalkbag"))

        if self.options.start_with_coffee:
            self.push_precollected(self.create_item("Coffee Unlock"))
```

Reply:
Thanks, I've added this, but ended up doing it myself as there were more places that this needed to be done